edge/pkg/metamanager/client/serviceaccount.go

```diff
@@ -54,6 +142,15 @@ func (c *serviceAccountToken) GetServiceAccountToken(namespace string, name stri
 	return handleServiceAccountTokenFromMetaManager(content)
 }
 
+func (c *serviceAccountToken) GetServiceAccountToken(namespace string, name string, tr *authenticationv1.TokenRequest) (*authenticationv1.TokenRequest, error) {
+	resource := fmt.Sprintf("%s/%s/%s", namespace, model.ResourceTypeServiceAccountToken, name)
+	tokenReq, err := getTokenLocally(resource, tr)
```

Comment:
for service account token、 secret、configmap， the volume plugin will periodiclly query the resource info, so we need a cache manager to reduce the query from sqlite.

Reply:
may be we can add a issue first to trace it

Reply:
Yes, Traced by https://github.com/kubeedge/kubeedge/issues/4144